packages/framework/src/types/schema.types/zod.schema.types.ts

```diff
@@ -3,7 +3,7 @@ import type zod from 'zod';
 /**
  * A ZodSchema used to validate a JSON object.
  */
-export type ZodSchema = zod.ZodType<Record<string, unknown>, zod.ZodTypeDef, Record<string, unknown>>;
+export type ZodSchema = zod.ZodType<unknown, zod.ZodTypeDef, unknown>;
```

Comment:
As with Json schema, each schema library allows for any schema to be used for simplicity, with centralized primitive handling in the base schema.